### PR TITLE
BE: Add field "videoUrl" and "previewImageUrl" to Post table

### PR DESCRIPTION
**Pull request by alexandr-bielan**
develop

## GitHub Board

**Issue link**

[ #204  ](https://github.com/ita-social-projects/dokazovi-be/issues/204#issue-829355423 )

## Code reviewers
- [x] @melnikovae87 

## Summary of issue

- [x]  Add field "videoUrl" and "previewImageUrl" to Post table using migrations
- [x]  Add new fields to the post entity and map into the DB

## Summary of change

- [ ]  Added fields "videoUrl" and "previewImageUrl" to Post table using migrations
- [ ]  Added new fields to the post entity and map into the DB
- [ ]  Fixed bug with date in  PostServiceImpl class

### CODE REVIEW

File: src/main/resources/db/migration/V1__init_db.sql [hunk 1]
```diff
@@ -133,6 +133,10 @@ CREATE TABLE POSTS
 ALTER TABLE POSTS
     OWNER TO POSTGRES;
 
+ALTER TABLE POSTS
+    ADD COLUMN VIDEO_URL VARCHAR,
```
Comment (andrewtyshkovets) [inline src/main/resources/db/migration/V1__init_db.sql:137]: @melnikovae87 Should we put default value "null" for this column? What's ur opinion?

Comment (andrewtyshkovets) [reply src/main/resources/db/migration/V1__init_db.sql:137]: If this field is optional for different typo of posts

File: src/main/java/com/softserveinc/dokazovi/entity/PostEntity.java [hunk 2]
```diff
@@ -49,6 +49,12 @@
 	@Column(columnDefinition = "TEXT")
 	private String preview;
 
+	@Column(columnDefinition = "TEXT", name = "video_url")
+	private String videoUrl;
+
+	@Column(columnDefinition = "TEXT", name = "preview_image_url")
```
Comment (melnikovae87) [inline src/main/java/com/softserveinc/dokazovi/entity/PostEntity.java:55]: Redundant

File: src/main/java/com/softserveinc/dokazovi/entity/PostEntity.java [hunk 3]
```diff
@@ -49,6 +49,12 @@
 	@Column(columnDefinition = "TEXT")
 	private String preview;
 
+	@Column(columnDefinition = "TEXT", name = "video_url")
```
Comment (melnikovae87) [inline src/main/java/com/softserveinc/dokazovi/entity/PostEntity.java:52]: Redundant. Just remove `columnDefinition = "TEXT"`

**Review by melnikovae87 — CHANGES_REQUESTED** [review]

**Review by melnikovae87 — APPROVED** [review]

**Review by andrewtyshkovets — APPROVED** [review]

**Pull request merged**